docs/docs/12-context.zh-CN.md

```diff
@@ -5,29 +5,29 @@ permalink: context-zh-CN.html
 prev: advanced-performance-zh-CN.html
 ---
 
-React最大的优势之一是很容易从你的React组件里跟踪数据流动。当你看着一个组件，你可以很容易准确看出哪个props被传入，这让你的APP很容易推断。
+React优势之一是你可以很容易地从React组件里跟踪数据流动。当你查看一个组件时，你可以很容易地判断出传入了哪些props，而这也有利于你的App进行逻辑推断。
 
-偶尔，你想通过组件树传递数据，而不在每一级上手工下传prop，React的 "context" 让你做到这点。
+有时，你想不通过在每一级组件设置prop的方式来向组件树内传递数据。那么，React的"context"特性可以让你做到这点。
 
-＞　注意：
-＞　
```

Comment:
Removing the empty ">" lines _might_ affect Jekyll's Markdown parsing. Could you build the docs (instructions [here](https://github.com/facebook/react/blob/master/docs/README.md)) to double check if the page renders correctly? Thanks! 😀